content/articles/isp-management-system/index.md

```diff
@@ -0,0 +1,786 @@
+### How to build an Internet Service Provider Management System using MERN Stack.
+
+In this tutorial, we will learn how to integrate our knowledge of various technologies, including MongoDB, Express, React, and Node.js, to create a functional web application.
+
+The web application is going to be a customer management system where one can create, read, update and delete customer information.
+
+Internet Service Providers (ISPs) offer internet access to customers. 
+The providers offer services including Wi-Fi installation for individual and corporate clients.
+The organizations need an informative and dynamic platform to manage their customers and ensure accurate business records. 
+You can learn more about ISPs [here](https://en.wikipedia.org/wiki/Internet_service_provider).
+
+The main aim of this tutorial is to provide foundational knowledge on how to create a MERN stack application through a real-life project.
+
+### Prerequisites
+
+1. An understanding of JavaScript.
+2. Basic knowledge of React and Node JS.
+3. Basic knowledge of MongoDB.
+4. You should also have [Node](https://nodejs.org/en/download/) installed on your machine.
+
+### Table of Contents
+
+- [Backend](#backend)
+- [Frontend](#frontend)
+- [Connecting Frontend to Backend](#Connecting-Frontend-to-Backend)
+- [Conclusion](#conclusion)
+
+### Backend
+The web application will follow the architecture shown below  consisting of the Client, Server, and Database.
+
+
+We will handle all the backend functionality of our app using a [REST API](https://www.smashingmagazine.com/2018/01/understanding-using-rest-api/) created using Node, Express, and MongoDB.
+- **MongoDB:** A database that uses a document-oriented data model.
+- **ExpressJS:** A web application framework for building web applications and APIs.
+- **Node.js:** A cross-platform JavaScript runtime environment used to build servers, and applications.
+
+#### Setting up Development Environment
+
+Let us first set up our development environment.
+Create a new folder and navigate to the new folder by running the following commands on a terminal
+
+``mkdir api``
+``cd api``
```

Comment:
You must use markdown formats for large code snippets. It will look something like this:

```
```bash
mkdir api
cd api
```
```